scripts/Reader.py.experimental

```diff
@@ -157,6 +189,8 @@ class Reader(object):
             if device_name == 'MFRC522':
                 self.reader = Mfrc522Reader()
             elif device_name == 'RDM6300':
+                #self.reader = Rdm6300Reader({'numberformat':'card_id_float'})
```

Comment:
do you intend to keep this commented code as example? If yes then an additional comment might help. Otherwise please remove the commented code.

Reply:
Yes, this was the Intention here, in order to Inform others about the possibilities. I will improve the comments.
I think there is no further documentation for this reader as the source or did I miss it?

I wanted to keep the change scope small, finaly this should end up somewhere in some configuration.
If there is a bigger interest, we can think about that for sure,